packages/desktop-client/src/components/accounts/Reconcile.tsx

```diff
@@ -131,9 +131,18 @@ export function ReconcileMenu({
     value: null,
     query: balanceQuery.query.filter({ cleared: true }),
   });
+  const lastSyncedBalance = account.balance_current;
   const format = useFormat();
+
   const [inputValue, setInputValue] = useState<string | null>(null);
 
+  // useEffect sets the input value to the clearedBalance when it is available
+  useEffect(() => {
```

Comment:
Why is this effect necessary? In general effects shouldn't be used to update state, only to interact with external systems. https://react.dev/learn/you-might-not-need-an-effect

Reply:
`clearedBalance` is not immediately available when this component bounds. I believe its because the `useSheetValue` uses a layout effect to perform the query for the value. We still want to use it as the default value when it is available, so we need the useEffect to set it to state